Cargo.toml

```diff
@@ -19,3 +19,8 @@ serde = "0.9"
 serde_derive = "0.9"
 serde_json = "0.9"
 tokio-core = "0.1"
+rpassword = "0.4.2"
```

Comment:
List this under `[dev-dependencies]` instead (it's a dependent of an example, but is not used in the main crate; c.f. http://doc.crates.io/specifying-dependencies.html)